src/Database/Table/Selection.php

```diff
@@ -227,6 +227,18 @@ public function fetch()
 
 
 	/**
+	 * Fetches single field.
+	 * @param  int
+	 * @return mixed|FALSE
+	 */
+	public function fetchField($column = 0)
+	{
+		$row = $this->fetch();
```

Comment:
As I see, it does not. So I can add `select()` as you proposed and make `$column` without default value. Or I can try to update `ActiveRow` to support indexed keys. What would you like reather?